Tentacle/Comment.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Comment.swift
+//  Tentacle
+//
+//  Created by Romain Pouclet on 2016-07-27.
+//  Copyright © 2016 Matt Diephouse. All rights reserved.
+//
+
+import Foundation
+import Curry
+import Argo
+
+public struct Comment: Hashable, CustomStringConvertible {
+
+    public let ID: String
+    public let URL: NSURL
+    public let createdAt: NSDate
+    public let updatedAt: NSDate
+    public let body: String
+    public let user: User
```

Comment:
Considering I spent some times trying to generate documentation, the irony of me forgetting to document my code is... 🙊